### ISSUE TITLE
Add costs for envelope efficiency improvement

### ISSUE THREAD

**Issue by robbieorvis** [Energy-Innovation/eps-us#82]
Right now, envelope improvements have no associated costs. We should add these in.

**Comment by jrissman** [comment]
The model structure is already in place for this calculation, and we have elasticity data already also.  The reason costs are zero is because we have zero spent on building components in `BASoBC BAU Amount Spent on Building Components`.  (We also have zero in that variable spent on "other components").  So it can't calculate a change if it doesn't know the baseline amount.

BAU amount spent on building envelope may be very high because that's a large part of the building, but the part of the envelope that is upgraded due to improved efficiency standards would only be a subset of that.  So maybe you just want the cost of insulation and window lites/panes in the `BASoBC` for envelope.

For "other components" (assuming you want to see non-zero costs for them too), you need to limit it to whatever energy-using components are not accounted for in the other building component categories.  This would include things such as escalators, elevators, pool pumps, etc.

This is a data-only issue, so I'm categorizing it that way in GitHub.

**Comment by jrissman** [comment]
I know it's hard to find good data for `BASoBC`.  I've thought about changing its format to be something like cost-per-square-foot of buildings affected by the standard.  That would be possible if we ever move to tracking buildings in units of floor area or similar, rather than just energy use quantities by building type and component type, as we do today.

**Comment by robbieorvis** [comment]
We may generally be able to find data that we could use to replace BASoBC by tracking the $/saved BTU from more efficient equpiment. 

For example, this paper has data on commercial building envelope improvements and the cost at different energy use intensities: https://www.mdpi.com/2071-1050/11/8/2444/pdf

For the US model, we have data from EIA on the costs of different technologies, and I can see a way we could translate BTUs shifted from efficiency to a single one time cost by using that data. 

What do you think about moving to this approach for a subsequent model release? 

For international data, we could simply adjust the cost multipliers by gdp per capita as a starting point. It would probably be more accurate than what we do now, anyway.

**Comment by jrissman** [comment]
Yes, I like the approach of replacing both `BASoBC` and the elasticity variable, `EoCPwEU Elasticity of Component Price wrt Energy Use`, with a single input variable that specifies the capital cost per BTU _per year_ saved from more efficient equipment.  This is similar to how we handle, for example, the capital costs of CCS equipment and of Direct Air Capture equipment.  It would also be more viable to leave $/BTU data in place, even without any GDP-per-capita adjustment, because it's a property of the costs of different types of equipment, and doesn't necessarily have to vary that much from country to country.  It would certainly reduce what is today a "high" priority variable to "medium" or even potentially to "low" priority.  I think your proposed change would be a substantial improvement.

**Comment by robbieorvis** [comment]
Okay, I’d be happy to compile the data for this. Sounds like it’s probably for a subsequent model release, but let’s plan on this for the next one.

________________________________
Robbie Orvis
Director of Energy Policy Design
Phone: 415-799-2171
98 Battery Street, Suite 202
San Francisco, CA  94111
www.energyinnovation.org<http://www.energyinnovation.org/>
[cid:image001.jpg@01D0D699.20A24470]
________________________________

Check out our new book, Designing Climate Solutions: A Policy Guide for Low-Carbon Energy<https://www.amazon.com/Designing-Climate-Solutions-Policy-Low-Carbon/dp/1610919564>
Available wherever books are sold

[Policy Design book cover]

**Comment by jrissman** [comment]
Sounds good.  Just send me the data any time, and I'll implement it pretty quickly after that.  Since the fix is going to change the structure, I removed the "data only" label.

**Comment by robbieorvis** [comment]
Adding a comment here that when we redo this data, we need to make sure we add the correct model structure and data to handle incremental capital costs from fuel shifting.
